### PR TITLE
Try to set-up new MacOS pipeline

### PR DESCRIPTION
**Pull request by ritzvik**

### CODE REVIEW

**Comment by ritzvik** [comment]
Mr @shreyasbapat , how do you like it?

**Comment by shreyasbapat** [comment]
Will review in a while

File: README.rst [hunk 1]
```diff
@@ -114,7 +111,6 @@ EinstienPy is usually tested on Linux, Windows and OS X on Python
 Platform        Site          Status
 ==============  ============  ===================
 Linux           CircleCI      |circleci|
-OS X            Travis CI     |travisci|
```
Comment (shreyasbapat) [inline README.rst]: Add the respective line here?

Comment (shreyasbapat) [reply README.rst]: https://github.community/t5/GitHub-Actions/Missing-documentation-on-GitHub-Actions-badges/m-p/30970#M628

File: .github/workflows/pythonpackage.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: Test_MacOS
+
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+    - 0.1.x
+    - 0.2.x
```
Comment (shreyasbapat) [inline .github/workflows/pythonpackage.yml]: Add this detail in the HOW-TO Release wiki page!

File: README.rst [hunk 3]
```diff
@@ -28,9 +28,6 @@
 .. |docs| image:: https://img.shields.io/badge/docs-latest-brightgreen.svg?style=flat-square
    :target: https://docs.einsteinpy.org/en/latest/?badge=latest
 
-.. |travisci| image:: https://img.shields.io/travis/einsteinpy/einsteinpy/master.svg?style=flat-square&logo=travis
```
Comment (shreyasbapat) [inline README.rst]: And the badge here as well

File: README.rst [hunk 4]
```diff
@@ -53,7 +50,7 @@
 
 |astropy| |mailing| |gitter| |riotchat| |license| |docs|
 
-|circleci| |travisci| |appveyor| |codecov| |codeclimate|
```
Comment (shreyasbapat) [inline README.rst]: And one badge here too

**Comment by ritzvik** [comment]
> If possible, create two well defined commits while removing travis, and adding github action. It will help if github decides to make this thing cost dollars :p

As you can see, that I already did.

File: .github/workflows/pythonpackage.yml (hunk 2 above)
Comment (ritzvik) [reply .github/workflows/pythonpackage.yml]: I don't think we have such a page :(

**Comment by ritzvik** [comment]
I'll remove the travis hook as well

File: .github/workflows/pythonpackage.yml (hunk 2 above)
Comment (ritzvik) [reply .github/workflows/pythonpackage.yml]: Open a respective issue about this 😕

**Pull request merged**